python/cudf/cudf/core/column/categorical.py

```diff
@@ -936,8 +936,9 @@ def ordered(self) -> Optional[bool]:
     def ordered(self, value: bool):
         self.dtype.ordered = value
 
-    def cat(self, parent: ParentType = None):
-        return CategoricalAccessor(self, parent=parent)
+    @property
+    def cat(self):
+        return CategoricalAccessor(self, parent=None)
```

Comment:
If this is a property how do we pass the parent to it?

Reply:
Apologies, this change was not needed. This was an intermediate change to facilitate an arrow metadata construction by passing our `Column` types. Reverting this as we can work-around this issue of metadata construction differently.